From da3b0246e3899e664f3b5a0be3c668ccf09f0692 Mon Sep 17 00:00:00 2001
From: Florin Sarbu <florin@balena.io>
Date: Fri, 3 May 2019 15:24:41 +0200
Subject: [PATCH] env_default.h: Add support for OS_KERNEL_CMDLINE

OS_KERNEL_CMDLINE is passed from meta-balena and contains cmdline arguments
for production images (supressing writing the boot log to console, no terminal
cursor blink etc).

Upstream-Status: Inappropriate [configuration]
Signed-off-by: Florin Sarbu <florin@balena.io>
---
 include/env_default.h | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/include/env_default.h b/include/env_default.h
index b18244f..d13388e 100644
--- a/include/env_default.h
+++ b/include/env_default.h
@@ -24,6 +24,9 @@ static char default_environment[] = {
 const uchar default_environment[] = {
 #endif
     RESIN_ENV
+
+"cmdline=" __stringify(OS_KERNEL_CMDLINE) "\0" \
+
 #ifndef CONFIG_USE_DEFAULT_ENV_FILE
 #ifdef	CONFIG_ENV_CALLBACK_LIST_DEFAULT
 	ENV_CALLBACK_VAR "=" CONFIG_ENV_CALLBACK_LIST_DEFAULT "\0"
-- 
2.7.4

